remo/profiles/tests/test_tasks.py

```diff
@@ -38,3 +41,31 @@ def test_invalid_date(self, mocked_now_date, mocked_reset_date):
 
         user_profile = UserProfile.objects.get(pk=user.userprofile.pk)
         eq_(user_profile.is_rotm_nominee, True)
+
+    def test_nominate_notification_base(self):
+        UserFactory.create(groups=['Mentor'])
+        subject = 'Nominate Rep of the month'
+        mentor_alias = settings.REPS_MENTORS_LIST
+
+        with patch('remo.profiles.tasks.now') as mocked_date:
```

Comment:
I would prefer this to be a decorator (mainly for consistency). Feel free to choose.